### PR TITLE
Rearrange some of the generated output to clear space for the new IA.

### PR DESCRIPTION
**Pull request by jpmedley**
R: @jeffposnick @addyosmani @gauntface

Fixes #*issue number*

*Description of what's changed/fixed.*

*Please ensure that `gulp lint test` passes locally prior to filing a PR!*

### CODE REVIEW

File: docs/modules_overview.md [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+  - [sw-runtime-caching](#sw-runtime-caching)
+- [External Contributions](#external-contributions)
+- [License](#license)
+
+## Ways to Use the Libraries
+
+Despite the number of modules in this package, there are a number of approaches to using these tools which many be used separately or together in any number of combinations.
+
+* **Precaching, runtime caching and routing**: Use [sw-lib](#sw-lib) to quickly
+  implement a [caching strategies](https://developers.google.com/web/fundamentals/instant-and-offline/offline-cookbook/) or a route.
```
Comment (jeffposnick) [inline docs/modules_overview.md]: There's article/noun disagreement in "a caching strategies".

File: docs/modules_overview.md [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+  below](#the-libraries).
+
+## Setting Up
+
+Each module is installed separately using the command line [as listed below](#the-libraries). To use a particular module:
+
+1. Install the module. For example:
+
+   `npm install --save-dev sw-lib`
+2. Copy the module's JavaScript and map files to your serving directory. To work properly these must be in the same directory as your service worker. For example:
```
Comment (jeffposnick) [inline docs/modules_overview.md]: These don't *need* to be copied into the same directory as your top-level service worker. They can be located anywhere that's within the root of your web site; you just need to pass in the correct URL when calling `importScripts()`.

Cf. https://github.com/GoogleChrome/sw-helpers/blob/master/packages/sw-routing/test/sw/express-route.js#L5

File: docs/modules_overview.md [hunk 3]
```diff
@@ -0,0 +1,228 @@
+---
+layout: index
+title: SW-Helpers
+navigation_weight: 0
+---
+
+<!-- DO NOT EDIT. This page is autogenerated. --><!-- To make changes, edit templates/README.hbs, not README.md! -->
```
Comment (jeffposnick) [inline docs/modules_overview.md]: Is this comment still accurate? If so, shouldn't the changes be made to the version of this file in `templates/`?

Comment (jpmedley) [inline docs/modules_overview.md]: Fixed.

File: docs/modules_overview.md (hunk 1 above)
Comment (jpmedley) [inline docs/modules_overview.md]: Fixed.

File: docs/modules_overview.md (hunk 2 above)
Comment (jpmedley) [inline docs/modules_overview.md]: Fixed.

File: templates/README.hbs [hunk 4]
```diff
@@ -1,4 +1,4 @@
-<!-- To make changes, edit templates/README.hbs, not README.md! -->
+<!-- To make changes, edit templates/README.hbs, not this file. -->
```
Comment (jeffposnick) [inline templates/README.hbs]: This file is `templates/README.hbs`, and it is what should be edited, so the "not this file" bit doesn't make sense.

How about: ""

Comment (jpmedley) [inline templates/README.hbs]: It will make sense in the file it appears in.

Comment (jeffposnick) [inline templates/README.hbs]: The comment also appears in `templates/README.hbs` and it doesn't make sense there. But `¯\_(ツ)_/¯`

**Pull request merged**